cegs_portal/search/views/v1/search.py

```diff
@@ -490,4 +510,4 @@ def get_data(self, options):
                 UserType.LOGGED_IN,
                 private_experiments=self.request.user.all_experiments(),
             )
-        return sig_reos
+        return Paginator(sig_reos, options["per_page"]).get_page(options["page"])
```

Comment:
Since you're paginating here the TSV response will not include any data. You should paginate in `def get` instead. and also update `_feature_sig_reg_effects.html` to handle pagination properly.